Assets/HoloToolkit-Examples/Prototyping/Shaders/UnlitTriplanar.shader

```diff
@@ -1,4 +1,10 @@
-﻿// Upgrade NOTE: replaced '_Object2World' with 'unity_ObjectToWorld'
+﻿// Upgrade NOTE: replaced 'UNITY_INSTANCE_ID' with 'UNITY_VERTEX_INPUT_INSTANCE_ID'
+
+// Upgrade NOTE: replaced 'UNITY_INSTANCE_ID' with 'UNITY_VERTEX_INPUT_INSTANCE_ID'
+
+// Upgrade NOTE: replaced 'UNITY_INSTANCE_ID' with 'UNITY_VERTEX_INPUT_INSTANCE_ID'
+
+// Upgrade NOTE: replaced '_Object2World' with 'unity_ObjectToWorld'
 
 Shader "HoloToolkit/UnlitTriplanar"
```

Comment:
Could we rename to `HoloToolkit/Examples/UnlitTriplanar`?

Reply:
Good point, will do.

Reply:
Also, I think the upgrade notes are injected by Unity, we can probably remove those as well, and add the default CLA license header.